src/main.rs

```diff
@@ -1,4 +1,133 @@
-fn main() {
-    println!("Hello, world!");
+use axum::{
+    async_trait,
+    extract::{FromRequest, Path, RequestParts},
+    handler::Handler,
+    http::StatusCode,
+    response::{IntoResponse, Json, Response},
+    routing::get,
+    Router, Server,
+};
+use serde_json::Value;
+use std::collections::HashMap;
+
+#[derive(Debug)]
+enum Version {
+    V1,
+}
+
+#[tokio::main]
+async fn main() {
+    let meta_routes = Router::new()
+        .route("/", get(root_handler))
+        .route("/ping", get(heartbeat_handler));
+
+    let transit_routes = Router::new()
+        .route("/bus", get(transit_bus_handler))
+        .route("/modes", get(transit_modes_handler));
```

Comment:
Nice! If we split the route handlers up in the future, this could be a method like `routes::transit::router() -> Router`.